src/renderer/corporate/member/corporate-add-member-container.jsx

```diff
@@ -48,17 +77,26 @@ class CorporateAddMemberContainer extends PureComponent {
 			jurisdiction: 'Please select a jurisdiction',
 			entityName: 'Please enter an entity name',
 			entityType: 'Please select a entity type',
-			creationDate: 'Please enter company creation date',
-			taxId: 'Tax id provided is invalid'
+			creationDate: 'Please enter company incorporation date',
+			taxId: 'Tax id provided is invalid',
+			type: 'Invalid member type',
+			positions: 'Invalid position',
+			country: 'Please select Residency',
+			nationality: 'Please select Nationality',
+			firstName: 'Please enter your first Name',
+			lastName: 'Please enter your last Name',
+			phoneNumber: 'Invalid phone number',
+			equity: 'Shares must be between 0 and 100',
```

Comment:
equity could be of different types, not necessary shares. 
`CorporateStructureSchema.getEquityForCompanyType` can be used for this

Reply:
Moving this fix to a different issue: #1865